src/translations/en.json

```diff
@@ -1701,7 +1701,14 @@
                 "save_button": "Save",
                 "currency": "Currency",
                 "edit_location": "Edit location",
-                "edit_location_description": "Location can be changed in zone settings"
+                "edit_location_description": "Location can be changed in zone settings",
+                "update_units_label": "Update unit of non temperature sensors",
```

Comment:
Isn't this title confusing if non temperature sensors are updated too?

Reply:
It looks like this:
![image](https://user-images.githubusercontent.com/14281572/215860229-1f3ae20e-aad3-4d29-a2bb-20d807faad4a.png)

Maybe it should say "Update the unit of all sensors" or something?

Reply:
If temperature sensors are also updated we should change it indeed. And let's remove the break after `... speed can be updated.`. It will be like this:

[x] Update unit of all sensors
The unit system has been changed, and the unit of some sensors like distance and speed can be updated. The unit of sensors where the unit has been edited from the UI or which can't be edited form the UI wil not be updated.

Note: The unit of temperature sensors is always updated

Reply:
Updated, the screenshots in the PR description show what it looks like now.